tests/js/client/chaos/test-module-chaos-load-cluster-nightly.js

```diff
@@ -0,0 +1,42 @@
+/* jshint globalstrict:false, strict:false, maxlen: 200 */
+/* global fail, print */
+
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2021 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License")
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Manuel Pöter
+////////////////////////////////////////////////////////////////////////////////
+
+const base = require("fs").join(require('internal').pathForTesting('client'), 'chaos', 'test-chaos-load-common.inc');
```

Comment:
I think this would be a good fit for test-helper.js

Next line is a debug print still to be removed?

Reply:
Thanks, I have removed the debug print.
> I think this would be a good fit for test-helper.js

I found an even better solution. By setting `process.env.NODE_PATH` before running the test we can allow `require` calls with relative paths.